pubspec.lock

```diff
@@ -473,4 +480,4 @@ packages:
     version: "3.1.0"
 sdks:
   dart: ">=2.12.0 <3.0.0"
-  flutter: ">=1.17.0"
+  flutter: ">=1.22.0"
```

Comment:
Why is this? Did you perhaps update your Flutter version? I'm still on 2.5.3, haven't updated yet, but I'll wait after the initial release and then update both Flutter and BLoC and test if everything is working.